src/assets/js/dom-elements.js

```diff
@@ -19,13 +19,15 @@ export const btnchecked = () => {
   const btn = document.createElement('button');
   btn.className = 'row-button button-checked btn-checkbox';
   btn.innerHTML = '&#10004;';
+  btn.name = 'checked';
   return btn;
 };
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be great if you add your own logo (Not Microverse logo) at the top of your readme file to make yourself a brand in the tech industry to catch the eyes of recruiters 👍